.github/workflows/licensed.yml

```
@@ -1,8 +1,8 @@
 name: Licensed
 
 on:
-  push: {branches: main}
-  pull_request: {branches: main}
+  push: {branches: [main]}
+  pull_request: {branches: [main]}
```

```suggestion
  push: 
    branches:
    - main
  pull_request:
    branches:
    - main
```

Should we just use the full syntax here?